stream.html

```diff
@@ -196,11 +196,7 @@
             teamlist[teamlist.length] = teams[key];
         }
         function comparescore(a,b) {
-            if (a.scores.league < b.scores.league)
-                return 1;
-            if (a.scores.league > b.scores.league)
-                return -1;
-            return 0;
+            return a.league_pos - b.league_pos;
```

Comment:
:+1: to doing this! :shakes-head:

Just to check: will this behave correctly when `league_pos` is not a clean number? Specifically -- actual ties after resolution are exposed as e.g: `7=`.